tests/wpt/web-platform-tests/domparsing/XMLSerializer-serializeToString.html

```diff
@@ -31,6 +31,11 @@ <h1>domparsing_XMLSerializer_serializeToString</h1>
   assert_equals(serialize(root), '<root><child1>value1</child1></root>');
 }, 'check XMLSerializer.serializeToString method could parsing xmldoc to string');
 
+test(function() {
+  var root = parse('<html><head></head><body><div></div><span></span></body></html>');
+  assert_equals(serialize(rootownerDocument), '<html><head/><body><div/><span/></body></html>');
```

Comment:
This is missing a `.` now 😆

Reply:
Sorry, seems like I'm wasting your time, is fixed now x.x